pl_examples/README.md

```diff
@@ -1,11 +1,67 @@
 # Examples   
 This folder has 3 sections:   
 
-### Domain templates   
-These are templates to show common approaches such as GANs and RL.   
+## Basic Examples   
+Use these examples to test how lightning works.   
+
+#### Test on CPU  
+```bash
+python cpu_template.py
+```
+
+---   
+#### Train on a single GPU
+```bash
+python gpu_template.py --gpus 1
+```   
+
+---    
+#### DataParallel (dp)   
+Train on multiple GPUs using DataParallel.
+
+```bash
+python gpu_template.py --gpus 2 --distributed_backend dp
+```   
+
+---
+#### DistributedDataParallel (ddp)    
+
+Train on multiple GPUs using DistributedDataParallel   
+```bash
+python gpu_template.py --gpus 2 --distributed_backend ddp
+```
 
-### Basic examples   
-These show the most common use of Lightning for either CPU or GPU training.   
+---
+#### DistributedDataParallel+DP (ddp2)    
 
-### Multi-node examples   
-These show how to run jobs on a GPU cluster using lightning.
\ No newline at end of file
+Train on multiple GPUs using DistributedDataParallel + dataparallel.
+On a single node, uses all GPUs for 1 model. Then shares gradient information
+across nodes.   
+```bash
+python gpu_template.py --gpus 2 --distributed_backend ddp2
+```
+
+## Multi-node example   
+
+This demo launches a job using 2 GPUs on 2 different nodes (4 GPUs total).
+To run this demo do the following:
+
+1. Log into the jumphost node of your SLURM-managed cluster.  
+2. Create a conda environment with Lightning and a GPU PyTorch version.   
+3. Choose a script to submit    
+
+#### DDP  
+Submit this job to run with distributedDataParallel (2 nodes, 2 gpus each)
+```bash
+sbatch ddp_job_submit.sh YourEnv
```

Comment:
It looks like these files were moved to be hidden. Do we need to change the sbatch call as well?

Reply:
Well it may not the desired thing to be hidden, have not realized on Linux this potential issue, will change it...